### ISSUE TITLE
[Desktop] Fix deps in //brave/browser/themes/

### ISSUE THREAD

**Issue by jumde** [brave/brave-browser#10656]
```
$ gn check .  //brave/browser/themes/*
ERROR at //brave/browser/themes/brave_theme_service.h:11:11: Include not allowed.
#include "chrome/browser/themes/theme_service.h"
          ^------------------------------------
It is not in any dependency of
  //brave/browser/themes:themes
The include file is in the target(s):
  //chrome/browser:browser
which should somehow be reachable.
___________________
ERROR at //brave/browser/themes/brave_theme_helper.h:10:11: Include not allowed.
#include "chrome/browser/themes/theme_helper.h"
          ^-----------------------------------
It is not in any dependency of
  //brave/browser/themes:themes
The include file is in the target(s):
  //chrome/browser:browser
which should somehow be reachable.
___________________
```

**Comment by simonhong** [comment]
@Gyuyoung This issue was closed by https://github.com/brave/brave-core/pull/6188 but it is not related with https://github.com/brave/brave-core/pull/6188. Re-opend.

**Comment by Gyuyoung** [comment]
Thanks for re-opening this issue. BTW, I don't know why this issue was closed by brave/brave-core#6188.
